geometry/optimization/graph_of_convex_sets.cc

```diff
@@ -377,7 +377,15 @@ std::string GraphOfConvexSets::GetGraphvizString(
     graphviz << " [label=\"" << e->name();
     if (result) {
       graphviz << "\n";
-      graphviz << "cost = " << e->GetSolutionCost(*result);
+      if (e->ell_.size() > 0) {
+        // SolveConvexRestriction does not yet return the rewritten costs.
+        if (result->get_decision_variable_index()->count(e->ell_[0].get_id()) !=
```

Comment:
This seems fine but I wonder why we don't combine the two boolean conditions:

```
if ((e->ell_.size() > 0) &&
      (result->get_decision_variable_index()->count(e->ell_[0].get_id()) !=
            0)) {
        // SolveConvexRestriction does not yet return the rewritten costs.
          graphviz << "cost = " << e->GetSolutionCost(*result);
      }
```
this also passes the unit test